### ISSUE TITLE
supporting private repos?

### ISSUE THREAD

**Issue by beeplin** [pd4d10/octohint#28]
Maybe I got something wrong, but it seems that octohint only works for public repos, not private ones?

**Comment by pd4d10** [comment]
Seems a token in query string is necessary to get private repo's code via https://raw.githubusercontent.com

Thanks for your report!

**Comment by beeplin** [comment]
Yes, and tools like `octotree` just guide users to create and input the access token.

**Comment by ran-eh** [comment]
Thanks for an amazing extension.  Is there a workaround to make this work, or should we wait patiently for someone to pick it up and fix it?  Cheers!
